vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -764,26 +769,26 @@ public Date getSvodPurchaseDate() {
     }
 
     /**
-     * Videos that are VODs are part of Seasons. Included on a Season
+     * Videos that are TVODs are part of Seasons. Included on a Season
      * Connection is the name of that season.
      *
      * @return the Season name, or null if this Video is not part of a Season
      */
     @Nullable
-    public String getSeasonName() {
+    public String getTvodSeasonName() {
         if (mMetadata != null && mMetadata.mConnections != null &&
```

Comment:
On second thought, maybe I do agree with it. Since the URI will undoubtably return a list of `Season`s, which currently is TVOD related only. If we do open it up in the future we will need to make a bunch of changes anyway. 👍